example/CountlyRNExample/Events.tsx

```diff
@@ -77,43 +37,28 @@ const startEvent = () => {
 */
 const timedEventWithSum = () => {
     // Event with sum
-    Countly.startEvent("timedEventWithSum");
-
-    const event: EventProps = {
-        eventName: "timedEventWithSum",
-        eventSum: "0.99",
-    };
+    Countly.events.startEvent("timedEventWithSum");
 
     setTimeout(() => {
-        Countly.endEvent(event);
+        Countly.events.endEvent("timedEventWithSum", undefined, 0.99);
     }, 1000);
 };
 
 const timedEventWithSegment = () => {
     // Event with segment
     Countly.startEvent("timedEventWithSegment");
 
-    const event: EventPropsCustom_1 = {
-        eventName: "timedEventWithSegment",
-        segments: { Country: "Germany", Age: "32" },
-    };
     setTimeout(() => {
-        Countly.endEvent(event);
+        Countly.events.endEvent("timedEventWithSegment", undefined, undefined, { Country: "Germany", Age: "32" });
     }, 1000);
 };
 
 const timedEventWithSumAndSegment = () => {
     // Event with Segment, sum and count
     Countly.startEvent("timedEventWithSumAndSegment");
 
-    const event: EventPropsCustom_1 = {
-        eventName: "timedEventWithSumAndSegment",
-        eventCount: 1,
-        eventSum: "0.99",
-        segments: { Country: "India", Age: "21" },
-    };
     setTimeout(() => {
-        Countly.endEvent(event);
+        Countly.events.endEvent("timedEventWithSumAndSegment", 1, 0.99, { Country: "India", Age: "21" });
     }, 1000);
```

Comment:
can you add a function (and its button) named "test_bad_values" in which you can record multiple events with bad values and lets see what happens

Reply:
done